### DISCUSSION TITLE
Error: JSON configuration file is in invalid JSON format!

### DISCUSSION THREAD

**Discussion by januridp** [hyperjumptech/monika#577] [Q&A] [answered]
Hi, there!
I got an error
<pre><font color="#55FF55"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> monika -c config.monika-whatsapp.example.yml 
 <font color="#AA0000">›</font>   Error: JSON configuration file is in invalid JSON format!
<font color="#FF5555"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> nano config.monika-whatsapp.example.yml 
<font color="#55FF55"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> monika -c config.monika-whatsapp.example.yml
 <font color="#AA0000">›</font>   Error: JSON configuration file is in invalid JSON format!
<font color="#FF5555"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> nano config.monika-whatsapp.example.yml
<font color="#55FF55"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> monika -c config.monika-whatsapp.example.yml
 <font color="#AA0000">›</font>   Error: JSON configuration file is in invalid JSON format!
<font color="#FF5555"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> code config.monika-whatsapp.example.yml 
<font color="#55FF55"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> monika -c config.monika-whatsapp.example.yml
 <font color="#AA0000">›</font>   Error: JSON configuration file is in invalid JSON format!
<font color="#FF5555"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> monika                                      
 <font color="#AA0000">›</font>   Error: Configuration file not found. By default, Monika looks for monika.json configuration file in the current directory.
 <font color="#AA0000">›</font>
 <font color="#AA0000">›</font>   Otherwise, you can also specify a configuration file using -c flag as follows:
 <font color="#AA0000">›</font>
 <font color="#AA0000">›</font>   monika -c &lt;path_to_configuration_file&gt;
 <font color="#AA0000">›</font>
 <font color="#AA0000">›</font>   You can create a configuration file via web interface by opening this web app: https://hyperjumptech.github.io/monika-config-generator/
<font color="#FF5555"><b>➜  </b></font><font color="#55FFFF"><b>config_sample</b></font> cd                                     
<font color="#55FF55"><b>➜  </b></font><font color="#55FFFF"><b>~</b></font> cd github/monika               
<font color="#55FF55"><b>➜  </b></font><font color="#55FFFF"><b>monika</b></font> monika -c monika.yml 
 <font color="#AA0000">›</font>   Error: JSON configuration file is in invalid JSON format!
<font color="#FF5555"><b>➜  </b></font><font color="#55FFFF"><b>monika</b></font> 
</pre>
somebody help me? I need a little guide here

Thanks in advance

**Comment by dennypradipta** [comment, marked as answer]
Hi @januridp thank you for creating a discussion!

Could you please run `monika -v` and provide your configuration (**please redact sensitive information such as emails, webhook URL, and API keys**) so that we know what version you're using and easier for us to troubleshoot?

**Reply by dennypradipta** [reply]
Following is an example of mine:
![image](https://user-images.githubusercontent.com/16670475/150077205-64fccfd8-a4d8-4e58-ac7f-d0b6e2ddc6c5.png)

My monika configuration:
```yaml
notifications:
  - id: 12f3e1df-3222-4a79-b931-146f964be711
    type: telegram
    data:
      group_id: <REDACTED>
      bot_token: <REDACTED>
probes:
  - id: 48c828bb-b761-41e7-b360-ba0692ded184
    name: ''
    requests:
      - url: https://github.com
        body: {}
        timeout: 10000
    incidentThreshold: 5
    recoveryThreshold: 5
    alerts: []
```

In case you have not updated your Monika yet, you can download the new version by using NPM/Yarn/download Monika binary from the release page.

**Reply by januridp** [reply]
Thanks, @dennypradipta , my prob was solved by updating monika to the latest version 1.6.x. My prev monika version is 1.3.x.

Terima kasih.

**Reply by dennypradipta** [reply]
Glad to be any of help! :smiley:
